README.md

```diff
@@ -72,15 +76,25 @@ COMMUTER_PATH_DELIMITER (optional, defaults to "/")
 COMMUTER_PORT (optional, defaults to 4000)
 ```
 
-Project uses [prettier](https://github.com/jlongster/prettier) for code formatting (`npm run format:code` and [package.json] (https://github.com/nteract/commuter/blob/master/package.json) has more options).
+Project uses [prettier](https://github.com/jlongster/prettier) for code formatting (`npm run format:code` and [package.json](https://github.com/nteract/commuter/blob/master/package.json) has more options).
 
 ## Tests
 
 1. `npm test`
 
 ## Deployment
 
-coming soon...
+There are few different ways to get commuter deployed on your severs:
+
+Firstly, to change default port `4000` set env variables `COMMUTER_PORT`
+
+1. If you want to deploy `master` branch then do `git clone git@github.com:nteract/commuter.git` and run `npm run start:production` on your server
+1. Currently, the [Demo](https://nteract-commuter.herokuapp.com/) app is deployed on [Heroku](https://www.heroku.com/).
+Instructions for getting started and setting up heroku-cli are [here](https://devcenter.heroku.com/articles/getting-started-with-nodejs#introduction).
+Once all **required env variables** are set do `git push heroku master`
+1. If you want to deploy npm published packages (**recomended**) then:
+  * Install commuter cli `npm install @nteract/commuter-cli -g`
+  * `exec commuter server` - the service is typically wrapped inside [daemontools](https://cr.yp.to/daemontools.html)
```

Comment:
It would be worth it to split this into two sections with the `###` triple heading -- "Deploying from master" and "Deploying from published packages". Whichever is recommended as the default should just be called "Deploying".